configs/test_network/test_client_config.json

```diff
@@ -0,0 +1 @@
+/home/app/Git/iroha/configs/client_config.json
```

Comment:
How this works?

Reply:
This is just symlink. I guess it should be `../client_config.json`